### PR TITLE
Makefile and prod web server

### PR DESCRIPTION
**Pull request by benadida**
To run this in production, a simple Makefile and node server setup that reuses `setupProxy.js`. All JavaScript for now because of this reuse of `setupProxy.js`. We can consider upgrading this all to Typescript when `setupProxy` supports it.

### CODE REVIEW

File: prodserver/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// This file runs a simple production web server of the React production build
+// it reuses setupProxy.js for consistent use of backend modules in dev and prod.
+// This is plain JS rather than our TS setup in the React app because of that.
+//
+/* eslint-disable */
```
Comment (beausmith) [inline prodserver/index.js]:
```suggestion
```

File: prodserver/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+/* istanbul ignore file */
+
+const express = require('express')
```
Comment (beausmith) [inline prodserver/index.js]:
```suggestion
import express from 'express'
```

File: prodserver/index.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+const proxy = require('../src/setupProxy')
```
Comment (beausmith) [inline prodserver/index.js]:
```suggestion
import proxy from '../src/setupProxy'
```

File: prodserver/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+const app = express()
+const port = 3000
+
+proxy(app)
+app.use('/', express.static('../build'))
+
+app.listen(port, () => console.log(`BMD listening on port ${port}!`))
```
Comment (beausmith) [inline prodserver/index.js]:
```suggestion
app.listen(port, () => console.log(`BMD listening on port ${port}!`)) // eslint-disable-line
```

**Comment by benadida** [comment]
@beausmith I believe I tried that approach you suggest, and it only works because `tsconfig.json` only includes `src`. The moment you try to have it include `prodserver`, it starts complaining about importing a file that uses a `require`, and I couldn't find a way around that. So, if you want to go this route, I can, but I think we're kinda fooling ourselves and we're not actually making `tsc` happy.

**Comment by beausmith** [comment]
Cool. Understood. I'm only trying to apply ESLint here as I don't see a reason not to lint this file. This suggesting has nothing to do with TypeScript.

**Comment by benadida** [comment]
after discussion in person with @beausmith, given difficulty of having node use modern import syntax, going to stick with what we have.

**Pull request merged**